fiaas_deploy_daemon/deployer/kubernetes/ready_check.py

```diff
@@ -62,6 +65,36 @@ def _ready(self):
                 dep.status.availableReplicas == expected_value and
                 dep.status.observedGeneration >= dep.metadata.generation)
 
+    def _is_certificate_ready(self, cert):
```

Comment:
Should we add the `notAfter` condition here?

Reply:
Sure thing! Thanks for the reminder